script.js

```diff
@@ -0,0 +1,32 @@
+console.log("script linked!");
+
+document.turn = "X";
```

Comment:
While you can add this information to `document`, I recommend adding to something specific to this problem domain, maybe `tictactoe.turn`?